client/components/view-other-account.jsx

```diff
@@ -42,6 +57,22 @@ export default class ViewOtherAccount extends React.Component {
     }
   }
 
+  updatingFollowerState() {
+    const requestObj = {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+        token: this.state.userLoggedIn.token
+      }
```

Comment:
I see that you are sending your token in a lot of requests, which is good if you want them to be private

But in your back end routes, like here
https://github.com/JihengNie/final-project/pull/22/files#diff-bcb729747c92e72c12e4590f736a334041a572c429756da8af60b6ab89c6be79R127

You are not actually making sure that the client is requesting **their own** resources, so rather than using a path parameter like `:accountId` on the server side, I recommend interrogating `req.user` instead (unless you want users to be able to see each other's resources.